vendor/bat-native-ledger/BUILD.gn

```diff
@@ -57,6 +57,21 @@ config("internal_config") {
   ]
 }
 
+config("endpoints_config") {
+  visibility = [
+    ":*",
+    "//brave/components/brave_adaptive_captcha:brave_adaptive_captcha",
```

Comment:
why is this needed in more places now? It can still be a public config

Reply:
Made it a public config.